### PR TITLE
Support block syntax for large numbers of transform rules.

### PR DESCRIPTION
**Pull request by shmookey**
Addresses https://github.com/CommBank/maestro/issues/384 by adding support for block-style assignment of transform rules.

### CODE REVIEW

File: maestro-macros/src/main/scala/au/com/cba/omnia/maestro/macros/Automap.scala [hunk 1]
```diff
@@ -215,7 +216,11 @@ object automap {
         
     /** Pull apart the source object, pad it out, put it back together. */
     def mkMapper(src: Tree): Tree = {
-      val q"def $name (..$srcArgs): $srcTo = (..$srcRules)" = src
+      val q"def $name (..$srcArgs): $srcTo = $srcBody" = src
+      val srcRules = srcBody match {
```
Comment (stephanh) [inline maestro-macros/src/main/scala/au/com/cba/omnia/maestro/macros/Automap.scala]: Codacy Issue found: [Default case not specified.](https://www.codacy.com/app/CommBank/maestro/file/634899072/issues/source?bid=2205397&fileBranchId=2309073#l220)

**Comment by stephanh** [comment]
So the difference between block and tuple style is using `{}` instead of `()` and no trailing `,`?

In that case can we just use the block style?

**Comment by shmookey** [comment]
@stephanh yep, done.

**Comment by rowandavies** [comment]
The syntax with blocks seems nicer, because no commas are required.  I guess multi-line expressions after the `:=` will now require parens or braces, right?

**Comment by shmookey** [comment]
@rowandavies yep

**Pull request closed without merging**